### DISCUSSION TITLE
Taskbar height and icon size not working after windows upgrade

### DISCUSSION THREAD

**Discussion by hailsatanonline** [ramensoftware/windhawk-mods#646] [Q&A]
Hi guys, the "taskbar height and icon size" not working after windows upgrade 23H2.

Is there anything that can be done to fix/workaround the situation, or just wait for an update to the Windhawk?

**Comment by m417z** [comment]
It was discussed here: https://github.com/ramensoftware/windhawk-mods/issues/640, https://github.com/ramensoftware/windhawk-mods/issues/641. Please refer to this blog post for details:
https://ramensoftware.com/windhawk-and-symbol-download-errors

**Reply by hailsatanonline** [reply]
Solved, thank you man!
